### PR TITLE
fix: Add instructions on how to disable Insights Management Component.

### PR DESCRIPTION
**Pull request by kaiwalyajoshi**
### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4463.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by cneth — APPROVED** [review]

File: pages/dkp/kommander/2.2/insights/insights-setup/index.md [hunk 1]
```diff
@@ -41,3 +41,27 @@ To receive insight items on the Insights Dashboard, enable the DKP Insights Engi
 1.  Enable the DKP Insights Engine on each Attached cluster from the DKP UI by selecting: **Applications** from within the relevant workspace, then select **Enable** from the vertical 3-dot menu in the bottom right of the Insights application tile.
 
    **Note**: The option for *Enable Another Instance* is not suggested as it is not supported in this technical preview release.
+
+## Disable DKP Insights Management
+
+
+The DKP Insights Management component can be disabled using the instructions for [configuring a Kommander installation.](../../install/configuration/)
+
+1. Initialize a default configuration file
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/insights/insights-setup/index.md:50]:
```suggestion
1. Initialize a default configuration file:
```

File: pages/dkp/kommander/2.2/insights/insights-setup/index.md [hunk 2]
```diff
@@ -41,3 +41,27 @@ To receive insight items on the Insights Dashboard, enable the DKP Insights Engi
...
+    ```
+    dkp install kommander --init > kommander.yaml
+    ```
+1. Delete or comment out the line containing `dkp-insights-management:null`
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/insights/insights-setup/index.md:54]:
```suggestion
1. Delete or comment out the line containing `dkp-insights-management:null`:
```

File: pages/dkp/kommander/2.2/insights/insights-setup/index.md [hunk 3]
```diff
@@ -41,3 +41,27 @@ To receive insight items on the Insights Dashboard, enable the DKP Insights Engi
...
+    ```
+    apiVersion: config.kommander.mesosphere.io/v1alpha1
+    kind: Installation
+    apps:
+      # Sections omitted...
+      dkp-insights-management: null # Delete or comment out this line.
+      # Sections omitted...
+    --
+    ```
+1. Install Kommander using the updated configuration file.
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/insights/insights-setup/index.md:64]:
```suggestion
1. Install Kommander using the updated configuration file:
```

File: pages/dkp/kommander/2.3/insights/insights-setup/index.md [hunk 4]
```diff
@@ -41,3 +41,27 @@ To receive insight items on the Insights Dashboard, enable the DKP Insights Engi
 1.  Enable the DKP Insights Engine on each Attached cluster from the DKP UI by selecting: **Applications** from within the relevant workspace, then select **Enable** from the vertical 3-dot menu in the bottom right of the Insights application tile.
 
    **Note**: The option for *Enable Another Instance* is not suggested as it is not supported in this technical preview release.
+
+## Disable DKP Insights Management
+
+
+The DKP Insights Management component can be disabled using the instructions for [configuring a Kommander installation.](../../install/configuration/)
+
+1. Initialize a default configuration file
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.3/insights/insights-setup/index.md:50]:
```suggestion
1. Initialize a default configuration file:
```

File: pages/dkp/kommander/2.3/insights/insights-setup/index.md [hunk 5]
```diff
@@ -41,3 +41,27 @@ To receive insight items on the Insights Dashboard, enable the DKP Insights Engi
...
+    ```
+    dkp install kommander --init > kommander.yaml
+    ```
+1. Delete or comment out the line containing `dkp-insights-management:null`
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.3/insights/insights-setup/index.md:54]:
```suggestion
1. Delete or comment out the line containing `dkp-insights-management:null`:
```

File: pages/dkp/kommander/2.3/insights/insights-setup/index.md [hunk 6]
```diff
@@ -41,3 +41,27 @@ To receive insight items on the Insights Dashboard, enable the DKP Insights Engi
...
+    ```
+    apiVersion: config.kommander.mesosphere.io/v1alpha1
+    kind: Installation
+    apps:
+      # Sections omitted...
+      dkp-insights-management: null # Delete or comment out this line.
+      # Sections omitted...
+    --
+    ```
+1. Install Kommander using the updated configuration file.
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.3/insights/insights-setup/index.md:64]:
```suggestion
1. Install Kommander using the updated configuration file:
```

**Review by gvasquezvargas — APPROVED** [review]
LGTM! Added a minor suggestion to facilitate reading and be consistent.

**Comment by kaiwalyajoshi** [comment]
@gvasquezvargas can you merge this PR? I'm not sure why Vale failed the 2nd time around as we didn't make any commits to the files with the errors.

**Comment by gvasquezvargas** [comment]
@kaiwalyajoshi vale was checking other files than the ones you had on your PR. We have seen this behaviour before, so nothing to worry. Just merged your PR. Thank you!

**Pull request merged**